agent/pkg/oas/output_rdwtyeoyrj.har.ldjson

```diff
@@ -0,0 +1,13 @@
+{"messageType": "http", "_source": null, "firstMessageTime": 1627298057.784151, "lastMessageTime": 1627298065.729303, "messageCount": 12}
```

Comment:
@undera should we merge these files to develop?

Reply:
Yes, there's no sensitive information inside